pkg/tsdb/elasticsearch/client/models.go

```diff
@@ -314,3 +314,56 @@ func (a *PipelineAggregation) MarshalJSON() ([]byte, error) {
 
 	return json.Marshal(root)
 }
+
+type pplresponse struct {
+	httpResponse *http.Response
+	reqInfo      *PPLRequestInfo
+}
+
+type PPLRequestInfo struct {
+	Method string `json:"method"`
+	Url    string `json:"url"`
```

Comment:
URL

In Go, abbreviations are capitalized.  See https://github.com/golang/go/wiki/CodeReviewComments#initialisms